README.md

```diff
@@ -1,19 +1,19 @@
 # react-cimpress-users
```

Comment:
If we want anyone to contribute anything it would be worth to list the steps that are needed to get this running from scratch. After checking out the repo I encountered a chain of errors:

I tried running `npm run start` but that failed because the translations weren't there. 
Since I knew what the error was, I ran `npm run build`, then that failed again because I didn't have `CLIENT_ID` and `CLIENT_SECRET` variables set. So I did set them and ran the build again. Only after that the `npm run start` succeeded.

Reply:
Added info in the README file.